masterlist.yaml

```diff
@@ -3823,6 +3819,11 @@ plugins:
           - 'Mythical Ages'
           - '[Mythical Ages - TrueStorms Merged Compatibility](https://www.nexusmods.com/skyrimspecialedition/mods/15617/)'
         condition: 'active("Mythical Ages.esp") and not active("Mythical_TS_Patch_.*\.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Obsidian Weathers and Seasons'
+          - '[Obsidian Weathers - TrueStorms Merged Compatibility](https://www.nexusmods.com/skyrimspecialedition/mods/17198/)'
+        condition: 'active("Obsidian Weathers.esp") and not active("Obsidian_TS_Patch_.*\.esp") and not active("True Storms - Obsidian Weathers - Patch .esp") and not file("Wander.esp")'
```

Comment:
This is also a duplicate, but I think moving it from `Obsidian Weathers.esp''s metadata for consistency is fine.
So you'd need to remove that duplicate message from `Obsidian Weathers.esp''s metadata.